src/views/v2/day/event/venue.php

```diff
@@ -24,7 +24,7 @@
 ?>
 <address class="tribe-events-calendar-day__event-venue tribe-common-b2">
 	<span class="tribe-events-calendar-day__event-venue-title tribe-common-b2--bold">
-		<?php echo isset( $venue_details['linked_name'] ) ? esc_html( $venue_details['linked_name'] ) : esc_html__( 'Venue Name', 'the-events-calendar' ); ?>
+		<?php echo isset( $venue_details['linked_name'] ) ? $venue_details['linked_name'] : esc_html__( 'Venue Name', 'the-events-calendar' ); ?>
```

Comment:
:warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$venue_details'

<sub>hash: 4a3f399e015c5de49dd9bed0d56631ab</sub>